plugins/Marketplace/stylesheets/plugin-details.less

```diff
@@ -43,9 +43,25 @@
     }
 
     #pluginDetailsTabs {
+        border-right: 1px solid silver;
         margin-top: 25px;
     }
 
+    // some plugins receive inline style from the marketplace
+    // override them for consistency
```

Comment:
The adjustment originally done in 6e02b6f9c1d52ee539e16e449bf4d28bde31ee27 targeted the new modal design.

Copied the full definition here with the flex/width adjustment. Most plugins receive the styling as inline in the marketplace "documentation" part (e.g. Funnels), though some don't (e.g. Cohorts). Now we should only check if that is a complete adjustment or if it breaks something unintended :frowning_face:

Reply:
@mneudert spoke with design on this yesterday and the longterm plan is to adjust the content to remove the left/right mechanism and side by side content with images as much as possible to make it better legible, so I think we don't need to spend too much time on that. I haven't found any other use of the styles within the codebase, so chances are high this is ok for now.